apps/api/test/services/test-database.service.ts

```diff
@@ -77,4 +77,113 @@ export class TestDatabaseService {
       await sql.end();
     }
   }
+
```

Comment:
all this would not work in CI env where there is no indexer db imported. should be solved differently. I got something locally gonna push a PR shortly.